packages/core/src/exports.ts

```diff
@@ -120,6 +120,14 @@ export function setUser(user: User | null): void {
   getIsolationScope().setUser(user);
 }
 
+/**
+ * The last event id of the isolation scope.
+ * @returns The last event id of the isolation scope.
+ */
+export function lastEventId(): string | undefined {
```

Comment:
l: I'd still add a warning that this works differently than expected

```suggestion
/**
 * The last error event id of the isolation scope.
 *
 * Warning: This function really returns the last recorded error event id on the current 
 * isolation scope.  If you call this function after handling a certain error and another error 
 * is captured in between, the last one is returned instead of the one you might expect. 
 * Also, ids of events that were never sent to Sentry (for example because 
 * they were dropped in `beforeSend`) could be returned.
 *
 * @returns The last event id of the isolation scope.
 */
export function lastEventId(): string | undefined {
```